tests/bluetooth/bsim_bt/bsim_test_iso/src/main.c

```diff
@@ -609,6 +671,52 @@ static void test_iso_recv_main(void)
 		FAIL("Local Host Terminate Failed.\n");
 	}
 
+#if defined(CONFIG_BT_CTLR_ISO_VENDOR_DATA_PATH)
```

Comment:
I would have put this sub-test in a separate function, but having everything here seems to follow the general style of this test, so ok.

Reply:
Yeah, good point. Just wanted to piggy-back on BIS setup already in place, but I think I'll move it

Reply:
On second thought, that requires a lot of redundancy in code and execution scripts. I think I'll leave it as is, if @cvinayak approves.

Reply:
Since I started the mess of having large executing test as a form of test driven development that kept adding the calls to APIs, I do not want to hinder this quick approach to validate a feature being implemented.

There has always been request to clean up the existing large test executions into smaller chunks, this has not been forgotten. I am sure we will always remember and when possibly time permitted improve in small portions.

I have no strong objection here.